arm_robots/src/arm_robots/robot.py

```diff
@@ -297,7 +299,7 @@ def follow_joint_config(self, joint_names: List[str], joint_positions, client: S
         trajectory.points.append(point)
         return self.follow_joint_trajectory(trajectory, client)
 
-    def follow_arms_joint_trajectory(self, trajectory: JointTrajectory, stop_condition: Optional[Callable] = None):
+    def follow_arms_joint_trajectory(self, trajectory: JointTrajectory, stop_condition: Optional[Callable] = None, group_name: Optional[str] = None):
```

Comment:
I'm starting to wonder why we can't just do use `move_group.execute` for all our robots...